### ISSUE TITLE
List validation rules?

### ISSUE THREAD

**Issue by burzum** [cakephp/docs#3373]
See this question: http://stackoverflow.com/questions/33014046/where-i-can-find-cakephp-3-validation-rules-table

> In laravel for example we have this page http://laravel.com/docs/5.1/validation#available-validation-rules

I agree that a table like this could be very useful.

**Comment by HavokInspiration** [comment]
There is [this paragraph](http://book.cakephp.org/3.0/en/core-libraries/validation.html#core-validation-rules) pointing to the [api doc](http://api.cakephp.org/3.0/class-Cake.Validation.Validation.html).

But I agree this can be hard to follow / read for a new user.

**Comment by dereuromark** [comment]
> The [API documentation] for the Validation class provides a good list of the validation rules that are available, and their basic usage.

I think it is not too hard to spot. The advantage of the API link is that the list is auto-up-to-date (when new ones are added or other renamed).
Whereas in the docs it can get out of sync.

**Comment by ravage84** [comment]
Simple solution:
Link from 
http://book.cakephp.org/3.0/en/core-libraries/validation.html#namespace-Cake\Validation
to
http://book.cakephp.org/3.0/en/core-libraries/validation.html#core-validation-rules

Something like
``This package provides __Core Validation Rules__``

**Comment by cake17** [comment]
I agree that it's not that easy to find core validation rules.
:1 for @ravage84 proposal, writing a link on top of the page.

**Comment by markstory** [comment]
I think this is fixed by #3385.
